ec/src/hashing/curve_maps/wb.rs

```diff
@@ -64,12 +65,16 @@ where
     }
 }
 
-/// Trait defining the necessary parameters for the WB hash-to-curve method
-/// for the curves of Weierstrass form of:
-/// of y^2 = x^3 + a*x + b where b != 0 but `a` can be zero like BLS-381 curve.
-/// From [\[WB2019\]]
+/// Trait defining the necessary parameters for the WB hash-to-curve method.
+///
+/// This method is used for curves in Weierstrass form defined by:
+///
+/// `y^2 = x^3 + a*x + b` where `b != 0`, but `a` can be zero,
+/// as seen in curves like BLS-381.
+///
+/// For more information, refer to [WB2019].
 ///
-/// - [\[WB2019\]] <http://dx.doi.org/10.46586/tches.v2019.i4.154-179>
+/// - [WB2019] <http://dx.doi.org/10.46586/tches.v2019.i4.154-179>
```

Comment:
We do want the square brackets around the reference

Reply:
fixed